src/features/products/ProductsPage.tsx

```diff
@@ -0,0 +1,79 @@
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+
+import ProductCard from './ProductCard';
+import carpet from '../../assets/carpet.png';
+import { seriesMock } from '../../assets/mockData/mockPoductData';
+
+export const initialStateMock = [
+  {
+    id: 0,
+    productName: '',
+    productDescription: {
+      pl: '',
+      en: '',
+    },
+    headDiameter: 0,
+    stickLength: 0,
+    weight: 0,
+    price: 0,
+    productImage: ``,
+    altText: {
+      en: '',
+      pl: '',
+    },
+    seriesId: 0,
+  },
+];
+
+const ProductsPage = () => {
+  const { i18n } = useTranslation('navAndFooter');
+  const [pending, setIsPending] = useState(true);
+  const [productObjectArray, setProductObjectArray] =
+    useState(initialStateMock);
+
+  useEffect(() => {
+    setTimeout(() => {
+      const getSeriesObject = async () => {
+        try {
+          // const response = await fetch();
+          const response = seriesMock;
+          if (!response) {
+            throw new Error();
+          } else {
+            setProductObjectArray(seriesMock.products);
+          }
+        } catch (error) {
+          console.log(error);
+        }
+        setIsPending(false);
+      };
+      getSeriesObject();
+    }, 1000);
+  }, []);
+
+  const seriesLang = seriesMock.seriesDescription;
+
+  return (
+    <>
+      {pending && <h1 className="text-center">Loading...</h1>}
```

Comment:
It's jumping in between the mocked "loading" but I guess it will be something to fix once the fetch from db will be performed

Reply:
Yes, its prepared for implementation of Loader and then we will fix it.